manage.py

```diff
@@ -251,6 +256,22 @@ def purge_user(email, actor):
     permanently_delete_user(email, actor=actor)
 
 
+@click.option('--token', '-t', help="token to lookup/describe")
```

Comment:
If the token is always required, this "option" might work better as [an argument](https://click.palletsprojects.com/en/master/arguments/#basic-arguments). Then `--token` or `-t` wouldn't be required